packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -19,6 +19,7 @@ export 'package:flutter_goldens_client/skia_client.dart';
 // https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package%3Aflutter
 
 const String _kFlutterRootKey = 'FLUTTER_ROOT';
+final RegExp _kReleaseBranch = RegExp(r'flutter-\d+\.\d+-candidate\.\d+');
```

Comment:
Instead of thinking this as release branches, what if we change it to only look at master|main? It seems unlikely we want to run goldens anywhere that isn't HEAD.

Reply:
To clarify, I mean checking `ENV['GIT_BRANCH'] == RegEx('master|main')`

Reply:
Oo, let me check a PR to see if this works. I think in presubmit checks it will reflect whatever the contributor named the branch.

Reply:
TIL! Looks like in post and presubmit the branch reflects `master`, that is way better - thank you!